CMakeModules/FindOpenCL.cmake

```diff
@@ -149,35 +117,13 @@ if(WIN32)
   endif()
 else()
   find_library(OpenCL_LIBRARY
-    NAMES OpenCL
-    PATHS
-        ENV LD_LIBRARY_PATH
```

Comment:
Not sure. I believe this is the official FindOpenCL from a newer version of CMake. I am sure we need to update this at some point but it seems to work on OSX and Debian 8. We can make changes later.